source/runbooks/set-up-macie.html.md.erb

```diff
@@ -0,0 +1,89 @@
+---
+owner_slack: "#modernisation-platform"
+title: Set Up Macie
+last_reviewed_on: 2024-04-18
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+# Introduction
+
+It is relatively easy to set up Macie for users and it need to be done on an account by account basis. **NOTE** Macie ONLY looks at S3 buckets and checks their access information so it will not look at databases or EC2 settings, for example.
+
+To do this you should connect to the environment on which you want to create Macie. Note, for accounts with 4 versions (development, test, preproduction, production) you will need to create Macie on each one. In theory you can just set it up in each account and allow one (say production) to use every account to access their data. At the time this document was produced this was not tested but can be used according to an AWS video showing a [demo](https://www.youtube.com/watch?v=8piwEQJJXdo).
+
+AWS also have a brief outline of the purpose of [Macie](https://docs.aws.amazon.com/macie/latest/user/what-is-macie.html) which can be checked to see if this is suitable.
+
+Starting Macie with Terraform is possible and the instructions are shown below. There is a pre-requisite that you need to ensure is completed before this is undertaken. I have also listed the details of setting up jobs. Although they can be set up (another example is shown) it may be beneficial to set these up manually. The example only shows a few of the many items that can be included.
```

Comment:
What is the pre-requisite?

Reply:
It's mentioned with the code. I will remove this here and try to highlight it a little more in the comments below